ADconnection.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bashes
```

Comment:
The shebang line `#!/bin/bashes` seems to be incorrect. It should be `#!/bin/bash` to specify the correct path to the bash interpreter.

Reply:
You are correct, my bad. Can I use something else?